doc/query.md

```diff
@@ -76,6 +81,54 @@ if (name == null) {
 
 If an index needs to be changed, first delete the existing index by calling `deleteIndexNamed(String indexName)` where the argument is the index name, then call  `ensureIndexed(List<Object> fieldNames, String indexName)` with the new definition.
 
+#### Managing a Text Index
+
+In order to perform text searches on documents in a datastore a text index needs to be created.  Our text indexing process leverages SQLite's full text search (FTS) functionality, therefore SQLite FTS needs to be available for a text index to be created.
```

Comment:
add comma: ".. in a datastore, a text index ...". 
break-up run-on sentence. "... (FTS) functionality. Therefore, SQLite FTS ... "
remove "for a text index to be created." -> "Therefore, SQLite FTS needs to be available."